sbi/inference/posteriors/mcmc_posterior.py

```diff
@@ -322,6 +323,8 @@ def _get_initial_params(
             num_chains: number of MCMC chains, generates initial params for each
             num_workers: number of CPU cores for parallization
             show_progress_bars: whether to show progress bars for SIR init
+            kwargs: Passed on to `_build_mcmc_init_fn`. Unused arguments should be
```

Comment:
what does "unused arguments should be absorbed mean"? will they be absorbed, or does the user need to do something about it?